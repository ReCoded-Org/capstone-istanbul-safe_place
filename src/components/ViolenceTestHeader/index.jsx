import React from "react";
import { Row } from "react-bootstrap";
import "./index.scss";

const ViolenceTestHeader = () => {
  return (
    <Row className="testHeader justify-content-md-center">
      <h1>Self Test</h1>
      <p>
        This test is developed by <br />
        <a href="https://dvs-snoco.org/what-is-domestic-violence-self-test/">
          Domestic Violence Services Snohomish County
        </a>
      </p>
    </Row>
  );
};

export default ViolenceTestHeader;
