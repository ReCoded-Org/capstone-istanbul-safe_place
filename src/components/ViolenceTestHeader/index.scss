@import "../../style/variables";

.testHeader {
  display: flex;
  flex-direction: column;
  text-align: center;
  margin-top: 2rem;
  font-family: $mainFontFamily;

  h1 {
    color: $primaryColor;
    padding-top: 0.8rem;
  }

  p {
    color: $darkColor;
  }

  a {
    color: $darkColor;
    text-decoration: underline;
  }
}
