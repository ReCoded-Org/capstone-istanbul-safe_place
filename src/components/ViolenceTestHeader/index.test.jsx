import React from "react";
import renderer from "react-test-renderer";
import ViolenceTestHeader from "./index";

it("renders correctly and matches the snapshot", () => {
  const tree = renderer.create(<ViolenceTestHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
