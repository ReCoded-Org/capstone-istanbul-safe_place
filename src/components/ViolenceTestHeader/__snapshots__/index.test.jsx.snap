// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and matches the snapshot 1`] = `
<div
  className="testHeader justify-content-md-center row"
>
  <h1>
    violenceTestPage.header
  </h1>
  <p
    dangerouslySetInnerHTML={
      Object {
        "__html": "violenceTestPage.subHeader",
      }
    }
  />
</div>
`;
